new/LIT7047Qene.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7047Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ba-zaman qadāmāwi ʾi-yǝḥǝmmi qʷāʿ</title>
+                <title xml:lang="en" corresp="#t1">In the old times the raven was not jealous</title>
```

Comment:
I would say: 

Ba-zaman qadāmāwi ʾi-yǝḥǝmmi qʷāʿ (Qǝne of wāzemā type)
In the old times the raven was not jealous ... 

(see below)